scripts/yourface.coffee

```diff
@@ -0,0 +1,40 @@
+# Description
+#   Your face is a hubot script. It replies with "your face"isms.
+#
+# Configuration:
+#   HUBOT_YOURFACE_PERCENT (optional)
+#		Percent chance that hubot will repond with a "Your face". Default is 40%
```

Comment:
I have to set the environment variables on deploy.
________________________________
Lydon Bergin / Senior Consultant
MCSD
615.224.0451 o
931.242.2520 c
251.706.4461 f

[Description: http://images.askcts.com/images/cts_logo_email.png]<http://www.askcts.com/>
www.askcts.com<http://www.askcts.com/>
lbergin@askcts.com<mailto:lbergin@askcts.com>

[Description: http://images.askcts.com/images/cts_logo_linkedin.png]<http://www.linkedin.com/company/cts-inc>[Description: http://images.askcts.com/images/cts_logo_twitter.png]<https://twitter.com/askCTS>[Description: http://images.askcts.com/images/cts_logo_facebook.png]<https://www.facebook.com/askCTS>[Description: http://images.askcts.com/images/cts_logo_googleplus.png]<https://plus.google.com/112055092176661665000/posts>

From: Robert N
Reply-To: lydonb/robort
Date: Tuesday, June 16, 2015 at 2:27 PM
To: lydonb/robort
Subject: Re: [robort] yourface, nlp stub, and conspiracy upgrades (#29)


In scripts/yourface.coffee<https://github.com/lydonb/robort/pull/29#discussion_r32560562>:

> @@ -0,0 +1,40 @@
> +# Description
> +#   Your face is a hubot script. It replies with "your face"isms.
> +#
> +# Configuration:
> +#   HUBOT_YOURFACE_PERCENT (optional)
> +#            Percent chance that hubot will repond with a "Your face". Default is 40%


Is that being set somewhere? It looks like it will just default to 40 if the env isn't defined

—
Reply to this email directly or view it on GitHub<https://github.com/lydonb/robort/pull/29/files#r32560562>.